readthedocs/projects/static-src/projects/js/import.js

```diff
@@ -157,14 +157,23 @@ function ProjectImportView (instance, config) {
 
     ko.computed(function () {
         var org = self.filter_org(),
-            orgs = self.organizations(),
```

Comment:
I removed this dependency because it was useless and produce hitting the API twice at init.